### DISCUSSION TITLE
Keyboard shortcuts on remote Windows agents

### DISCUSSION THREAD

**Discussion by AndersMogensenMJBR** [Ylianst/MeshCentral#5666] [Q&A] [answered]
Is it possible to send all keyboard shortcuts to the remote agent, or is it a browser thing that some is not send?

**Comment by si458** [comment, marked as answer]
sadly its a browser thing, because the browser takes over control of some of the shortcuts, so some work, some dont
but you can use the send keyboard shortcuts option, bottom left to help you, right click the text box and customize what commands you use the most

![Screenshot 2024-01-05 115241](https://github.com/Ylianst/MeshCentral/assets/765314/ce78c476-f359-40a6-a051-b6238460996c)
![Screenshot 2024-01-05 115249](https://github.com/Ylianst/MeshCentral/assets/765314/c5ba442e-2b0d-4205-9f94-575fc5e952d1)
![Screenshot 2024-01-05 115258](https://github.com/Ylianst/MeshCentral/assets/765314/f9b3fdba-f55a-42f3-bd5b-0023f6a579ee)

**Reply by AndersMogensenMJBR** [reply]
Thanks for your help :-)
